src/biogeochem/SatellitePhenologyMod.F90

```diff
@@ -24,7 +24,8 @@ module SatellitePhenologyMod
   private
   !
   ! !PUBLIC MEMBER FUNCTIONS:
-  public :: SatellitePhenology     ! CLMSP Ecosystem dynamics: phenology, vegetation
+  public :: GetSatellitePhenologyInputs ! put the data into the correct format
```

Comment:
I kind of think of a method with Get in the title as a getter method. And this is more than that. We tend to use Calc or Update for something more extensive. This is really where the interpolation is applied though.

So maybe something like...

CalcSatellitePhenologyTimeInterp?